### ISSUE TITLE
Visualize bbox3d

### ISSUE THREAD

**Issue by no-Seaweed** [garrickbrazil/M3D-RPN#40]
Hi,

I am writing the code for visualizing both GT and predict 3d BBox on validation set images by multiplying the output 3d coordinates with the corresponding calibration matrix. Your output matches the GT, but both of those 2 projected bboxes are a bit off to the actual car. What the information I am missing? Here is the example I have.

![image](https://user-images.githubusercontent.com/40819442/87918571-3d173100-ca3c-11ea-9c01-a05726e3243f.png)

Best,
Shukai

**Comment by githubAccR** [comment]
Did you ever find out how to fix this?

**Comment by JianWu313** [comment]
@no-Seaweed  can you send me a skript only for the prediction,I just wanna used the pretrained weights for inference,thanks in advanve

**Comment by Colin131** [comment]
in "rpn_util.py" function "test_kitti_3d"
delete line: 
> y3d += h3d/2

then you will be fine

**Comment by Sondosmohamed1** [comment]
@no-Seaweed  could you illustrate how did you visualize the result ? i want to reimplement the algorithm using camera  data  i retrained it but  i do not know how to implement it using new camera data(for example png /jpg image ) .
